### DISCUSSION TITLE
New page?

### DISCUSSION THREAD

**Discussion by ExperienceRO** [new-sankaku/stable-diffusion-webui-simple-manga-maker#34] [Q&A] [answered]
Hello...
First of all, thank you for the amazing work.
I've tryed to access discord, but, I wasn't able to find a channel to post... 😅
Anyway..
Just to understand it weel and all.
I've finished a panel (page), so I want to go to next page. Do I need to download the actual page or there's a way to add a new a page an then save the whole project as one single thing?

For example, this is the page one:
![image](https://github.com/user-attachments/assets/3b8eb31a-6701-4095-900b-b3704245c94c)

Now I want to start the page 2, how to??

If there's no way, can I suggest this option?? I have no clue how much work does it need to add such thing, but, ask doesn't hurt does it?? 😆😆😆

Thank you.

**Comment by new-sankaku** [comment]
At the moment, pages are created one at a time.
Therefore, you need to either go to File and select Project Save or Image Download.
After that, you need to select another Panel to create a new page.

I'm thinking of making it multi-page compatible at some point, but I don't have the time yet...
If you need to have multiple pages open, I think the only way to do this at the moment is to open multiple pages in browser tabs.

**Reply by ExperienceRO** [reply]
I see...
If not too much to ask, what about a vertical continuous panel like we use in Webtoon??
Yeah, I know, I can do it in photoshop and all or even on Krita, but, you know, we always look for ways to make things easier... 😅
Once again, thank you a lot for your effort and work, it's absolutely amazing, I share about the manga maker every time I can, hope it helps..
Thank You.. 🥰

**Comment by new-sankaku** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/3392d543-ecf4-4618-9d3a-8d21785695fd)

Thank you! I hope more people use it!

For Webtoon, you can create vertical images from this custom panel.

However, there are some problems.

Because it creates very large vertical images, please use the ZOOM function.

Also, if the size is too large, the operation may become slow as the number of images and objects increases. Therefore, it is necessary to divide it into a certain size.

![image](https://github.com/user-attachments/assets/b01946ad-b7d3-416b-af33-65508caf5153)
